src/prolly/leaf.rs

```diff
@@ -83,6 +83,11 @@ impl Leaf {
         }
     }
 
+    pub fn len(&self) -> usize {
+        let root = leaf::get_root_as_leaf(self.chunk.data());
+        root.entries().map(|v| v.len()).unwrap_or(0)
```

Comment:
I think it is safe to unwrap `entries()`. We never leave it put.

Reply:
You are right, fixed in https://github.com/rocicorp/repc/pull/239.